src/lib/annotations/doc/DocAnnotator.js

```diff
@@ -384,6 +384,14 @@ class DocAnnotator extends Annotator {
      * @return {void}
      */
     renderAnnotationsOnPage(pageNum) {
+        // Force any annotation canvases on page to clear & re-render
+        const pageEl = docAnnotatorUtil.getPageEl(this.annotatedElement, pageNum);
+        const annotationLayerEl = pageEl.querySelector(`.${CLASS_ANNOTATION_LAYER_HIGHLIGHT}`);
```

Comment:
This isn't actually clearing the canvas, it's just removing the canvas from the DOM. You can either use canvas's `clearRect()` or set the `width` and or `height` (I've never been a fan of this method as an explicit clear call makes most sense to the user/reader. But it works!)

Reply:
Changed this logic to re-scale rather than remove and add again